.github/workflows/lock.yml

```
@@ -13,7 +13,7 @@ jobs:
     if: github.repository_owner == 'sphinx-doc'
     runs-on: ubuntu-latest
     steps:
-      - uses: dessant/lock-threads@v2
+      - uses: dessant/lock-threads@v3
         with:
           github-token: ${{ github.token }}
           issue-lock-inactive-days: '30'
```

This configuration was renamed since v3.
https://github.com/dessant/lock-threads/blob/master/CHANGELOG.md

```suggestion
          issue-inactive-days: '30'
```

---

`pr-lock-inactive-days` was also renamed to `pr-inactive-days`.